src/lighteval/utils.py

```diff
@@ -200,16 +200,9 @@ def is_tensorboardX_available() -> bool:
 )
 
 
-def is_openai_available() -> bool:
-    return importlib.util.find_spec("openai") is not None
-
-
-NO_OPENAI_ERROR_MSG = "You are trying to use an Open AI LLM as a judge, for which you need `openai`, which is not available in your environment. Please install it using pip."
-
-
 def can_load_extended_tasks() -> bool:
     imports = []
-    for package in ["langdetect", "openai"]:
```

Comment:
Why did you change this? We still don't want openai as a native dep of ligtheval

Reply:
I was remove openai entirely from the codebase, i'm adding it back as we might need to do comparaisons between results.

Reply:
ok, so as optional dep here should be good then